src/misc/lv_area.h

```diff
@@ -265,6 +260,10 @@ void lv_area_align(const lv_area_t * base, lv_area_t * to_align, lv_align_t alig
 void lv_point_transform(lv_point_t * p, int32_t angle, int32_t scale_x, int32_t scale_y, const lv_point_t * pivot,
                         bool zoom_first);
 
+void lv_point_array_transform(lv_point_t * p, size_t p_count, int32_t angle, int32_t scale_x, int32_t scale_y,
```

Comment:
Can we have meaningful parameter names here, too?
`p` -> `points`
`p_count` -> `count`

(same for `lv_point_transform` function)